pages/public/signup.vue

```diff
@@ -360,3 +388,16 @@ export default {
   animation: nono 300ms, intro paused;
 }
 </style>
```

Comment:
São não esquecer de pôr o `scoped `todas as vezes que um componente novo for criado
`<style scoped> ...`